TripHouse/index.html

```diff
@@ -189,15 +187,83 @@ <h2 class="homes-guests-text">Homes guests loves</h2>
             <p class="cities-text">Berlin, Germany</p>
           </li>
         </ul>
-        <svg class="button-homes" width="48" height="48">
+        <a class="button-homes-link" href="">
+          <svg class="button-homes" width="48" height="48">
+            <use href="../images/svg-sprite.svg#Button-homes"></use>
+          </svg>
+        </a>
+      </div>
+    </section>
+    <section class="best-dest-cont">
+      <h2 class="best-dest-heading">The best destinations</h2>
+      <div class="best-dest-but-cont">
+        <button class="best-dest-but" type="submit">Regions</button>
+        <button class="best-dest-but" type="submit">Cities</button>
+        <button class="best-dest-but" type="submit">Places of interest</button>
+        <button class="best-dest-but" type="submit">Places</button>
+      </div>
+      <ul class="best-dest-list">
+        <li class="best-dest-item Maldives">
+          <div class="maldives-wrapper">
+            <img class="pic1" src="../images/maldives-pic.svg" alt="Maldives" />
+            <button class="book-now" type="submit">Book now</button>
+          </div>
+          <p class="best-dest-text">Maldives</p>
+        </li>
+        <li class="best-dest-item Spain">
+          <img src="../images/spain-pic.svg" alt="Spain" />
+          <p class="best-dest-text">Spain</p>
+        </li>
+        <li class="best-dest-item Norway">
+          <img src="../images/norway-pic.svg" alt="Norway" />
+          <p class="best-dest-text">Norway</p>
+        </li>
+        <li class="best-dest-item Island">
+          <img src="../images/island-pic.svg" alt="Island" />
+          <p class="best-dest-text">Island</p>
+        </li>
+      </ul>
+      <a class="button-dest-link" href="">
+        <svg class="button-dest" width="48" height="49">
+          <use href="../images/svg-sprite.svg#button-dest"></use>
+        </svg>
+      </a>
+      <a class="button-dest-right-link" href="">
+        <svg class="button-dest-right" width="48" height="48">
           <use href="../images/svg-sprite.svg#Button-homes"></use>
         </svg>
-      </div>
+      </a>
+    </section>
+    <section class="sign-up-cont">
+      <a class="close-link" href="">
```

Comment:
use button for better semantic. you should see the difference between link and button
[read more](https://htmlacademy.ru/blog/boost/frontend/links-or-buttons)